frontend/src/app/commonComponents/MultiSelect/MultiSelectDropdown/MultiSelectDropdown.test.tsx

```diff
@@ -692,7 +693,7 @@ describe("MultiSelectDropdown component", () => {
         />
       );
 
-      fireEvent.click(screen.getByTestId("multi-select-input"));
+      user.click(screen.getByTestId("multi-select-input"));
```

Comment:
using fireevent for the click here was breaking the tests so I swapped it with the higher-level user click

Reply:
userEvent methods are missing `await`.